### DISCUSSION TITLE
suggestions for an arrow marker whose tip is at the initial / terminal datum?

### DISCUSSION THREAD

**Discussion by JacobBeaudry** [airbnb/visx#1376] [Q&A]
I'd like to have an arrow marker (on a line-series) whose point is located at the exact coordinate of the initial / terminal datum, using the simplest possible solution. 

I tried making a custom marker, adjusting refX / refY; however, there are various limitations caused by the orientation of the marker that limit the effectiveness of this kind of a solution. I ended up adding a stroke outline to hide the part of the curve that is visible, underneath the marker: 
![image](https://user-images.githubusercontent.com/75229506/142772339-5a2e772d-94c9-40d8-9ba2-957188518294.png)
Which is okay, but not great. The line is still visible, & the stroke color is hard-coded, so doesn't match the background, in general: 
![image](https://user-images.githubusercontent.com/75229506/142772373-ceae99be-f88d-49aa-ae02-63b25eaaace9.png)
Any suggestions for a better implementation / solution?

**Comment by williaster** [comment]
Hi @RaxAdaam, I think overall the `@visx/marker` package might do what you want, here's a demo https://airbnb.io/visx/curves
![image](https://user-images.githubusercontent.com/4496521/148141011-e224d32f-e459-467d-ac1c-59c3461e13ca.png)

If not, can you be more specific about which packages you're using and possibly provide a link to the examples you have images of? (it's tricky to make good suggestions with limited info)

**Reply by JacobBeaudry** [reply]
Hi again, Chris -- thanks for the reply. We put something (better than the above) in place, temporarily, as we're preparing for a beta launch and just needed something. Our programmers already have a very full plate ATM, so I'll follow-up on this after the launch. Thanks for your time and help!
